### DISCUSSION TITLE
Email vs. github Discussions

### DISCUSSION THREAD

**Discussion by arthurrge** [aiaa-deic/computation-ai#17] [General]
### Situation: email is being used for discussion rather than github

Problems w/ using github include: 
- 
- People need to set up their github account and log in
- Need to go to github site to read / respond

Problems w/ using Email include:
- 
- No archive of discussions to help newly-joining people to catch up
- Difficulties searching, retaining for reference 
- Distribution list gets out of synch with group membership (managed on github)

Problems w/ trying to use github but discussions actually use Email include:
- 
- github hides email addresses, which makes it difficult to know if everyone added to github is included in the email to/cc list
- (so we manage the group in github but may inadvertently exclude people from discussions)
Given that the github site is "public" 
- I'm hesitant to even list our email addresses on github (to be scraped outside the scope of good actors)
- While we rely on email to converge, **_please let me know if you did not receive the email I sent this morning_** (1-May-2024 Subject: DEIC COMP+AI Aligning)

### Desirable State: active discussions within github Discussion area

**Comment by opinonfischer** [comment]
Good morning Rick, I did not received the email mentioned above. ~Olivia
